Commands/durabletask/scheduler/_create.md

```diff
@@ -0,0 +1,16 @@
+# [Command] _durabletask scheduler create_
+
+Create a Scheduler
+
+## Versions
+
+### [2024-10-01-preview](/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcmVzb3VyY2Vncm91cHMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5kdXJhYmxldGFzay9zY2hlZHVsZXJzL3t9/2024-10-01-preview.xml) **Stable**
+
+<!-- mgmt-plane /subscriptions/{}/resourcegroups/{}/providers/microsoft.durabletask/schedulers/{} 2024-10-01-preview -->
+
+#### examples
+
+- Create a scheduler in eastus
+    ```bash
+        durable-task scheduler create --resource-group testrg --scheduler-name testscheduler --location eastus --ip-allowlist "[0.0.0.0/0]" --sku "{capacity: 1, name:Dedicated, redundancyState: None}" --tags "{}"
```

Comment:
`redundancyState` must not be included in any requests.

Also I think the SKU parameter is formatted incorrectly.